README.md

```diff
@@ -434,3 +434,10 @@ None.
 ## Author Information
 
 [HIFIS Software Team](https://software.hifis.net)
+
+## Contributors
+
+We would like to thank and give credits to the following contributors of this
+project:
+
+* Be the first to be named here!
```

Comment:
Wouldn't it be better to list the contributors in a separate file and link to it and the contributing guide here?

Reply:
I would actually prefer to list it below the author information in the README to make the list more prominent.